Makefile

```diff
@@ -235,19 +228,24 @@ CLEAN_TARGETS += clean-vendor
 clean-vendor:
 	-rm -rf $(VENDOR_DIR)
 
-CLEAN_TARGETS += clean-glide-cache
-.PHONY: clean-glide-cache
-## Removes the ./glide directory.
-clean-glide-cache:
-	-rm -rf ./.glide
-
-$(VENDOR_DIR): glide.lock glide.yaml
-	$(GLIDE_BIN) --quiet install --strip-vendor
-	touch $(VENDOR_DIR)
-
 .PHONY: deps
 ## Download build dependencies.
-deps: $(VENDOR_DIR)
+deps: $(DEP_BIN) $(VENDOR_DIR)
+
+# install dep (see https://golang.github.io/dep/docs/installation.html)
+$(DEP_BIN):
+	@echo "Installing 'dep' at $(DEP_BIN)"
+	@mkdir -p $(DEP_BIN_DIR)
+ifeq ($(UNAME_S),Darwin)
+	curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-darwin-amd64 -o $(DEP_BIN)
+else
+	curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $(DEP_BIN)
+endif
```

Comment:
@xcoulon can you please include a checksum check like we're doing for go?

```sh
RUN test -n $USE_GO_VERSION_FROM_WEBSITE \
    && cd /tmp \
    && wget --no-verbose https://dl.google.com/go/go1.10.linux-amd64.tar.gz \
    && echo "b5a64335f1490277b585832d1f6c7f8c6c11206cba5cd3f771dcb87b98ad1a33  go1.10.linux-amd64.tar.gz" > checksum \
    && sha256sum -c checksum \
    && tar -C /usr/local -xzf go1.10.linux-amd64.tar.gz \
    && rm -f go1.10.linux-amd64.tar.gz
```

Reply:
ok

Reply:
@xcoulon thank you for addressing this in dcfd73c